src/main/java/main/internet/shop/security/filter/AuthorisationFilter.java

```diff
@@ -0,0 +1,83 @@
+package main.internet.shop.security.filter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import main.internet.shop.lib.Injector;
+import main.internet.shop.model.User;
+import main.internet.shop.model.role.UserRole;
+import main.internet.shop.service.UserService;
+
+public class AuthorisationFilter implements Filter {
```

Comment:
```suggestion
public class AuthorizationFilter implements Filter {
```

let's also refactor AuthenticationFilter as proposed in the checklist